core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -766,14 +766,13 @@ class BrokerServer(
         CoreUtils.swallow(socketServer.shutdown(), this)
       if (brokerTopicStats != null)
         CoreUtils.swallow(brokerTopicStats.close(), this)
```

Comment:
`BrokerTopicStats` extends `Closeable` now, so you can apply `Utils.closeQuietly`, right?

Reply:
Exactly. Let me update this one.

Reply:
Apply this one in commit fb93c6b.